src/test/javascript/spec/pipe/artemis-date.pipe.spec.ts

```diff
@@ -33,17 +26,13 @@ describe('ArtemisDatePipe', () => {
     it('Return empty string if given date time is null', () => {
         let localizedDateTime = pipe.transform(null);
         expect(localizedDateTime).to.be.equal('');
-        localizedDateTime = pipe.transform(moment(null));
+        localizedDateTime = pipe.transform(dayjs(null));
```

Comment:
This would remove the purpose of the test to explore correct behavior is null is passed. I instead added another test testing for undefined. Ironically this test failed, since dayjs(undefined) returns the current date, so I removed it again

Reply:
I don't think this test is very good then because we don't need to test the null behaviour of datejs.

Can you check what the actual result of `datejs(null)` is? `undefined`?
Because I would just check `pipe.transform()` with whatever that value is. That makes more sense in my opinion.

Reply:
Agreed. According to https://day.js.org/docs/en/parse/now, its just invalid and therefore the same as the other test. I just left the direct calls to the pipe.

Reply:
Nice, thanks.